### DISCUSSION TITLE
Using https instead of nats protocol

### DISCUSSION THREAD

**Discussion by TobyDarmawan** [nats-io/nats.py#548] [Q&A]
I'm trying to send a request to the health endpoint (i.e. 8222/healthz), but the servers currently uses `nats://` instead of `https://`, which is a protocol not supported by curl. How would I send a request like:
`curl "https://<hostname>:8222/healthz"` instead of `curl "nats://<hostname>:8222/healthz"`

**Comment by wallyqs** [comment]
8222 port is for the http protocol based monitoring endpoint, to get the same results via NATS you need to use a system account user and then send a request to `$SYS.REQ.SERVER.PING.HEALTHZ`, so something like

```python
nc = await nats.connect("sys:sys@endpoint:4222"
inbox = nc.new_inbox()
sub = await sub.subscribe(inbox)
await nc.publish("$SYS.REQ.SERVER.PING.HEALTHZ", b'', reply=inbox)

# Check responses received by servers
for i in range(0,3):
  msg = await sub.next_msg()
  print(msg)
```
